src/pages/manager/CustomersListPage.js

```diff
@@ -3,9 +3,63 @@ const { expect } = require('@playwright/test');
 export class CustomersListPage {
   constructor(page) {
     this.page = page; 
+    this.lastRowLocator = page.getByRole('row').last();
+    this.firstNameCellLocator = this.lastRowLocator.getByRole('cell').nth(0);
```

Comment:
There is a typo in the variable name `firstNameCellLocator`. Previously, it was mentioned as `firstNamaCellLocator`. Ensure consistency across your codebase.